netbox/extras/plugins/__init__.py

```diff
@@ -41,6 +46,23 @@ class PluginConfig(AppConfig):
     # Caching configuration
     caching_config = {}
 
+    def ready(self):
+
+        # Register navigation menu items (if defined)
+        register_menu_items(self.verbose_name, self.get_menu_items())
+
+    def get_menu_items(self):
```

Comment:
Rather than having an author potentially override a method, what if we just let them specify the module name and have it default to `navigation`? This is similar to how django handles the models module https://docs.djangoproject.com/en/3.0/ref/applications/#django.apps.AppConfig.models_module

Reply:
That's fine too. Do you want to make it an attribute of PluginConfig?

Reply:
Yeah, that's what I would do. Something like `navigation_module = 'navigation'`?